### DISCUSSION TITLE
On MacOS where do the exported codes get stored?

### DISCUSSION THREAD

**Discussion by cosmicqbit** [ente-io/ente#196] [General]
![image](https://github.com/ente-io/auth/assets/75565639/e9c84c15-61ae-46ed-a3c1-47b966998cf9)

The ente auth does ask for the confirmation while exporting the codes but it doesn't show where the codes are exported exactly, be it encrypted or plain text. I had same issue on iPadOS, couldn't locate the exported file.

**Comment by ua741** [comment]
Thanks for reporting; this will be fixed in the next release."

**Reply by cosmicqbit** [reply]
That sounds great. Thanks for taking note of this ❤️

**Reply by ua741** [reply]
Fixed in v1.0.54
